onebusaway-api-webapp/src/main/java/org/onebusaway/api/impl/TripsConfigurationServiceClientFileImpl.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (C) 2015 Cambridge Systematics, Inc.
```

Comment:
No, CS didn't do this.  Please don't copy and paste headers without understanding them